packages/node_modules/@webex/plugin-meetings/test/unit/spec/members/request.js

```diff
@@ -118,5 +118,25 @@ describe('plugin-meetings', () => {
         assert.equal(requestParams.body.hand.raised, true);
       });
     });
+
+    describe('#lowerAllHands', () => {
+      it('sends a PATCH to the locus endpoint', async () => {
+        const locusUrl = url1;
+        const memberId = 'test1';
+
+        const options = {
+          requestingParticipantId: memberId,
+          locusUrl,
+        };
+
+        await membersRequest.lowerAllHandsMember(options);
```

Comment:
issue: The return value should be checked (other code will want to use the promise from the request, so we should ensure it is returned).